Gems/ScriptCanvas/Code/Editor/Framework/ScriptCanvasReporter.inl

```diff
@@ -62,7 +63,7 @@ namespace ScriptCanvasEditor
         }
 
         m_isReportFinished = true;
-        Bus::Handler::BusDisconnect(m_scriptCanvasId);
+        Bus::Handler::BusDisconnect(m_graph);
```

Comment:
GraphIdentifier and other direct references to component or entity systems from within the ScriptCanvas runtime have been replaced by references to the ExecutionState. When required by specific graphs, the EntityId/RuntimeComponent are stored in the user data of the ExecutionState. Determining this need is discovered and saved at graph compile time.